src/java.base/windows/native/libnio/ch/wepoll.c

```diff
@@ -156,7 +156,7 @@ WEPOLL_EXPORT int epoll_wait(HANDLE ephnd,
 #define WIN32_LEAN_AND_MEAN
 
 #undef _WIN32_WINNT
-#define _WIN32_WINNT 0x0600
+#define _WIN32_WINNT 0x0601
```

Comment:
This is 3rd party code and would prefer not change it if possible.

Reply:
I Alan, I agree (thats why I did not change the define in harfbuzz,  but I missed that wepoll.c is 3rd party code as well).

Reply:
Yes sure I did it .